sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1107,6 +1107,14 @@ object functions {
    */
   def input_file_name(): Column = withExpr { InputFileName() }
 
+  /**
+   * Return true iff the column is Infinity.
+   *
+   * @group normal_funcs
+   * @since 1.6.0
```

Comment:
Need to fix these versions, here and elsewhere. This change would land in Spark 2.4.0.